### DISCUSSION TITLE
Changing the url "docs" to something else

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#60] [Q&A] [answered]
I would like to showcase my projects with this theme.

How can I ensure that the URLs have `/projects` instead of `/docs`?

**Comment by colinwilson** [comment, marked as answer]
You'll need to create the following file structure & templates in your Hugo site:

```treeview
layouts
└── projects
    ├── baseof.html
    ├── list.html
    └── single.html
```

Then place you content in:

```treeview
content
└── projects
    └── _index.md
```

You could then navigate to `https://yoursite.com/projects`

The [Hugo Template Guide](https://gohugo.io/templates/) has a lot more extensive and detailed info on building templates

**Reply by ghost** [reply]
Thanks Colin!

I also changed `hugo.toml` to have:

```
[menu]
[[menu.primary]]
    name  = "Projects"
    url = "projects/"
    identifier = "projects"
    weight = 10
```

and it worked out.
